src/Mailman.Services/Google/GmailServiceImpl.cs

```diff
@@ -22,17 +30,83 @@ internal class GmailServiceImpl : GoogleServiceBase, IEmailService
         }
 
 
-        public async Task SendEmailAsync(string to, 
-            string cc, 
-            string bcc, 
+        public async Task SendEmailAsync(
+            IEnumerable <string> to, 
+            IEnumerable <string> cc, 
+            IEnumerable <string> bcc, 
             string subject, 
             string body)
-        {
+        {   
+        
             using (var gmailservice = await _googleSheetsServiceAccessor.GetGmailServiceAsync())
-            {
-                throw new NotImplementedException();
+            {    
+                Boolean noReciver = true;
+                System.Text.Encoding.RegisterProvider(System.Text.CodePagesEncodingProvider.Instance);
+                var msg = new MailMessage() {
+                    Subject = subject,
+                    Body = body,
+                    IsBodyHtml = true
+                };
+
+                foreach (var address in to)
+                {
+                    if (!string.IsNullOrEmpty(address)){
+                        noReciver = false;
+                        msg.To.Add(new MailAddress(address));
+                    }
+                }
+                foreach (var address in cc)
+                {                    
+                    if (!string.IsNullOrEmpty(address)){
+                        noReciver = false;
+                        msg.CC.Add(new MailAddress(address));
+                    }
+                }
+                foreach (var address in bcc)
+                {
+                    if (!string.IsNullOrEmpty(address)){
+                        noReciver = false;
+                        msg.Bcc.Add(new MailAddress(address));
+                    }
+                }
+                //foreach (string path in attachments)
+                //{
+                //    Attachment attachment = new Attachment(path);
+                //    msg.Attachments.Add(attachment);
+                //}
+
+                if (noReciver != true)  
+                {
+                    MimeKit.MimeMessage mimeMessage = MimeMessage.CreateFromMailMessage(msg);
+                    Message message = new Message();
+                    message = new Message {Raw = Base64UrlEncode(mimeMessage.ToString())};
```

Comment:
nit: these 2 lines can be combined into one. i.e.:
`Message message = new Message() { Raw = … }`